builder/patches/rkmpp/0000-fix-vpx-collisions.patch

```diff
@@ -0,0 +1,154 @@
+From 154a74b0a75eaecd5022cef3cb520ee949b578d9 Mon Sep 17 00:00:00 2001
+From: Brett Jia <dev.bjia56@gmail.com>
+Date: Wed, 19 Jun 2024 11:44:23 -0400
+Subject: [PATCH] namespace vars that conflict with vpx

```

Comment:
Perhaps submitting this upstream? https://github.com/rockchip-linux/mpp/issues/580
Maintaining mpp patches in jellyfin-ffmpeg repo is not ideal for me.